src/lib/ui/PersonaInput.svelte

```diff
@@ -47,6 +49,9 @@
 		<button type="button" on:click={create} disabled={status === 'pending'}> Create </button>
 	</form>
 </Markup>
+<CenteredBlock>
+	<Message icon="‼">your persona name is visible in your communities</Message>
```

Comment:
```suggestion
	<Message icon="‼">your persona name is visible to others</Message>
```
Not 100% on if this is the right phrase, but the other one felt a little awkward